src/features/CourtCaseDetails/Tabs/Panels/Offences/Offence/HearingResult.tsx

```diff
@@ -6,6 +6,17 @@ import { TableRow } from "../../TableRow"
 import pleaStatus from "@moj-bichard7-developers/bichard7-next-data/dist/data/plea-status.json"
 import verdicts from "@moj-bichard7-developers/bichard7-next-data/dist/data/verdict.json"
 
+enum DurationUnit {
```

Comment:
I wonder if it would be better to add this enum to https://github.com/ministryofjustice/bichard7-next-data/tree/main/output-data/types

Reply:
We did consider that. Woul be worth doing it for Gender and RemandStatus codes as well.